renovate.json

```diff
@@ -1,5 +1,22 @@
 {
   "extends": [
     "config:base"
+  ],
+  "packageRules": [
+    {
+      "packagePatterns": ["eslint"],
```

Comment:
shouldn't there be ` "@hydrant/eslint-config", "eslint-plugin-ava"` be added here?

Reply:
i think the pattern should match everything with eslint in the name

Reply:
👍